invariance.partial.R

```diff
@@ -0,0 +1,144 @@
+invariance.partial <- function(model, type, MI.cut=5, min.anchor=3, write.model=T, write.pattern=T, name.model=NULL, 

```

Comment:
Mam se do tohodle voodoo poustet / budes to jeste nekdy v budoucnu pravidelne vyuzivat? :)

Reply:
Ne, to je historie.